qiskit/circuit/library/arithmetic/__init__.py

```diff
@@ -0,0 +1,21 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The arithmetic circuit library."""
+
+from .integer_comparator import IntegerComparator
+from .linear_pauli_rotations import LinearPauliRotations
+from .piecewise_linear_pauli_rotations import PiecewiseLinearPauliRotations
+from .polynomial_pauli_rotations import PolynomialPauliRotations
+from .weighted_adder import WeightedAdder
```

Comment:
this file hasn't been added, and also i think you're missing an import of `FunctionalPauliRotations`

Reply:
Ah, yes, I wanted to postpone the `WeightedAdder` it needs some tests. Thanks!